doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -4537,9 +4538,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We will create a subclass of `az.SamplingWrapper`. Therefore, instead of having to implement all functions required by `az.reloo` we only have to implement `sel_observations` (we are cloning `sample` and `get_inference_data` from the `PyStanSamplingWrapper` in order to use `apply_ufunc` instead of assuming the log likelihood is calculated within Stan). \n",
+    "We will create a subclass of {class}`~arviz.SamplingWrapper`. Therefore, instead of having to implement all functions required by {func}`~arviz.reloo` we only have to implement {func}`~arviz.SamplingWrapper.sel_observations` (we are cloning {func}`~arviz.PyStanSamplingWrapper.sample` and {func}`~arviz.SamplingWrapper.get_inference_data` from the {class}`~arviz.SamplingWrapper` in order to use `apply_ufunc` instead of assuming the log likelihood is calculated within Stan). \n",
```

Comment:
{func}`~arviz.PyStanSamplingWrapper.sample` to {func}`~arviz.SamplingWrapper.sample`

Reply:
Changed them